app/views/applicants/unassigned.html.erb

```diff
@@ -11,11 +11,11 @@
               <p>Applied <%= pluralize(@applicant_presenter.days_since_last_applied(app.applied_on), "day", "days") %> ago</p>
             </div>
             <span class="applicant-actions">
-              <%= link_to "Auto-Assign", {:controller => 'applicants', :action => 'assign_craftsman', :id => app.id }, {:class => "button primary"} %>
               <%= form_for :applicant_to_assign, url: specify_craftsman_path, method: :post do |f| %>
-                <%= f.select(:chosen_crafter, options_for_select(@craftsmen.map { |crafter| [crafter['name'], crafter['name']] })) %>
-                <%= f.hidden_field(:id, :value => app.id) %>
-                <%= f.submit "Assign to Crafter", class: "button primary" %>
+                <label class="dropdown-label">Assign to:</label>
+                <%= f.select(:chosen_crafter, options_for_select(@craftsmen.map { |crafter| [crafter['name'], crafter['name']] }.unshift(["Available Crafter","auto"])), {}, class: "dropdown-height dropdown") %>
```

Comment:
I think you can just map name and it'll set it as both without having to make the array for the label/value.